docs/examples.md

```diff
@@ -331,10 +331,83 @@ object CSVPrinter extends IOApp.Simple {
 @:@
 
 
+## Parsing and transforming raw data
+This example was written by [Thanh Le] for the [scalachess library](https://github.com/lichess-org/scalachess/pull/352).
+
+@:select(scala-version)
+
+@:choice(scala-3)
+
+```scala mdoc:silent
```

Comment:
I'll add a reset here too, just to be sure that this runtime won't get contaminated by previous snippets